packages/server/lib/cloud/artifacts/upload_artifacts.ts

```diff
@@ -16,6 +16,14 @@ import { NetworkError } from '../network/network_error'
 
 const debug = Debug('cypress:server:cloud:artifacts')
 
+const rmWspAndTrim = (str: string) => {
```

Comment:
Does `rmWsp` stand for `removeWhitespace`? If so, suggest just spelling it out:

```suggestion
const removeWhitespaceAndTrim = (str: string) => {
```